mw-webapp/src/component/table/columns.tsx

```diff
@@ -1,4 +1,8 @@
-import {CellContext, ColumnDef, createColumnHelper} from "@tanstack/react-table";
```

Comment:
What was changed in this file? Didn't found(

Reply:
perhaps formatting